life-and-letters/modernized/05-chapter-4.adoc

```diff
@@ -195,9 +195,15 @@ Thomas Kite.
 
 Allowed anger to prevail, and,
 under its influence, spoke unadvisedly.
-Oh! for more of the disposition of the Lamb!'``' In the Fourth month, 1812,
-Thomas Kite being in Trenton, his father wrote to him:
-"`Susanna Home`'s meeting was large;
+Oh!--for more of the disposition of the Lamb!
+
+[.offset]
++++[+++In the Fourth month, 1812, Thomas Kite being in Trenton, his father wrote to him:]
+
+[.small-break]
+'''
```

Comment:
The offset on line 200 is good, but since an offset creates spacing above and below it, it's not necessary to put a small break either directly before or directly after a small break, as it makes the spacing double, and slightly too large.  So let's remove the small break here.